db/models/review.js

```diff
@@ -4,9 +4,18 @@ const {TEXT, FLOAT} = require('sequelize')
 
 module.exports = db => db.define('reviews', {
   rating: {
-    type: FLOAT
+    type: FLOAT,
+    allowNull: false,
+    validate: {
+      isNumeric: true
+    }
   },
   comment: {
     type: TEXT
   }
 })
+
+module.exports.associations = (Review, {Magnet, User}) => {
```

Comment:
A question that I don't know the answer to: We definitely want the association between Magnet and User for review stored on REVIEW, not in join tables?